aemedge/blocks/header/header.css

```diff
@@ -256,17 +256,17 @@ header nav .nav-sections .nav-top>ul li:first-child {
     letter-spacing: 0;
   }
 
-  header nav .nav-sections .default-content-wrapper > ul > li > ul > li {
+  header nav .nav-sections .default-content-wrapper>ul>li>ul>li {
     padding: 8px 0;
```

Comment:
I really don't like the removal of the spaces in these, can you change your linter rules?